### PR TITLE
chore: removed brightcove package and dependencies

### PR DESCRIPTION
**Pull request by Daniel-Georgiev**
Removed brightcove player and it's dependencies

### CODE REVIEW

File: packages/article-magazine-standard/__tests__/web/__snapshots__/semantic.web.test.js.snap [hunk 1]
```diff
@@ -138,7 +138,7 @@ exports[`1. a full article 1`] = `
       target="_blank"
     >
       <svg
-        aria-label="[title]"
+        aria-label="icon-twitter"
```
Comment (aronshamash) [inline packages/article-magazine-standard/__tests__/web/__snapshots__/semantic.web.test.js.snap]: I think this is for a different PR no ?

Comment (Daniel-Georgiev) [reply packages/article-magazine-standard/__tests__/web/__snapshots__/semantic.web.test.js.snap]: Yep, it's not meant to go inside this one, will revert once I have the chance

Comment (Daniel-Georgiev) [reply packages/article-magazine-standard/__tests__/web/__snapshots__/semantic.web.test.js.snap]: Reverted.

**Comment by aronshamash** [comment]
should it be a `BREAKING CHANGE` @ValGeorgiev ?

**Comment by ValGeorgiev** [comment]
> should it be a `BREAKING CHANGE` @ValGeorgiev ?

I think the title of the PR was not correct, we are not deprecating brightcove, we are removing not used package `brightcove-video`. We are still using brightcove, but in video package directly. There is no need of this package and its not used. BREAKING CHANGE will bump all packages that are using brigthcove-video package with 1 major version. Currently there are no packages that are using brigthcove-video (only several mocks, but these are not dependancies), so I don't think this is needed in that case. I will merge it

**Pull request merged**